### ISSUE TITLE
aria-hidden on selected choice graphic?

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/quizful#36]
I'm not sure why `aria-hidden` is used here.

`<li><i class="fa fa-times" aria-hidden="true"></i><p>Red</p></li>`

> aria-hidden: Indicates that the element and all of its descendants are not visible or perceivable to any user as implemented by the author.

These elements are not decoration, they signify the selected answer. So why hide them from screen readers? With the change to radio inputs suggested in https://github.com/Thinkful-Ed/quizful/issues/35 though it won't matter because then the options would be semantic and accessible as native inputs.
